source/type/type_utils.h

```diff
@@ -49,4 +50,13 @@ const type::StopTime& earliest_stop_time(const std::vector<type::StopTime>& sts)
 
 pbnavitia::RTLevel to_pb_realtime_level(const navitia::type::RTLevel realtime_level);
 
+namespace type {
+
+enum class DirectionType { All = 0, Forward, Backward };
```

Comment:
Would this enum better be in `type/type.h` ?

Reply:
Because there is no more data struct into type.h, since @kinnou02's changes :smile: 
But I hesitated to create a `direction_type.h`. What do you think ? One header for all **direction_type** stuff :crown:

Reply:
If we keep the proto enum... Why not use it directly and remove that one?